Sources/Orbit/Components/Collapse.swift

```diff
@@ -0,0 +1,183 @@
+import SwiftUI
+
+/// Hides long or complex information.
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/collapse/)
+public struct Collapse<Header: View, Content: View>: View {
+
+    struct _Collapse: View {
+        let header: Header
+        let headerVerticalPadding: CGFloat
+        let content: Content
+        @Binding var isExpanded: Bool
+
+        var body: some View {
+            VStack(alignment: .leading, spacing: 0) {
+                SwiftUI.Button {
+                    withAnimation(.easeInOut(duration: 0.2)) {
+                        isExpanded.toggle()
+                    }
+                } label: {
+                    HStack(spacing: 0) {
+                        header
+                            .accessibility(.collapseHeader)
+                        Spacer(minLength: .xSmall)
+                        Icon(.chevronDown)
+                            .rotationEffect(isExpanded ? .degrees(180) : .zero)
+                    }
+                    .padding(.vertical, headerVerticalPadding)
+                }
+                .buttonStyle(CollapseButtonStyle())
+
+                if isExpanded {
+                    content
+                        .padding(.vertical, .small)
+                }
+            }
+            .overlay(Separator(), alignment: .bottom)
+        }
+    }
+
+    let header: Header
+    let headerVerticalPadding: CGFloat
+    let content: Content
+    let isExpanded: Binding<Bool>?
+    @State var internalIsExpanded = false
+
+    public var body: some View {
+        _Collapse(
+            header: header,
+            headerVerticalPadding: headerVerticalPadding,
+            content: content,
+            isExpanded: isExpanded ?? $internalIsExpanded
+        )
+    }
+}
+
+public extension Collapse {
+
+    /// Creates Orbit Collapse component with a binding to the expansion state.
+    init(isExpanded: Binding<Bool>, @ViewBuilder content: () -> Content, @ViewBuilder headerContent: () -> Header) {
+        self.header = headerContent()
+        self.headerVerticalPadding = 0
+        self.content = content()
+        self.isExpanded = isExpanded
+    }
+
+    /// Creates Orbit Collapse component.
+    init(@ViewBuilder content: () -> Content, @ViewBuilder headerContent: () -> Header) {
```

Comment:
We could still keep the `isExpanded: Bool = false` property in this init for a one-way direction.

Reply:
We could, but not even Apple does that for their components. I wouldn't overcomplicate it.